tests/unit/plugins/inventory/test_flightctl.py

```
@@ -1,8 +1,15 @@
+import json
 import unittest
-from unittest.mock import patch, MagicMock
+from unittest.mock import patch, MagicMock, PropertyMock
```

Flake8 (7.3.0)

[error] 'unittest.mock.PropertyMock' imported but unused

(F401)